endpoint/http/src/main/java/de/fraunhofer/iosb/ilt/faaast/service/endpoint/http/request/mapper/DeleteAllAssetLinksByIdRequestMapper.java

```diff
@@ -21,32 +21,26 @@
 import de.fraunhofer.iosb.ilt.faaast.service.model.request.DeleteAllAssetLinksByIdRequest;
 import de.fraunhofer.iosb.ilt.faaast.service.util.EncodingHelper;
 import de.fraunhofer.iosb.ilt.faaast.service.util.IdentifierHelper;
+import java.util.Map;
 
 
 /**
  * class to map HTTP-DELETE-Request path: lookup/shells/{aasIdentifier}
  */
 public class DeleteAllAssetLinksByIdRequestMapper extends RequestMapper {
 
-    private static final HttpMethod HTTP_METHOD = HttpMethod.DELETE;
-    private static final String PATTERN = "^lookup/shells/(.*)$";
+    private static final String AAS_ID = "aasId";
```

Comment:
Might be useful to create a static helper class or move to super class for such constants. Same for `submodelId`, `submodelelementId`, `conceptId` and `packageId`